docs/data.rst

```diff
@@ -0,0 +1,239 @@
+by NCAR's `Computational and Information Systems Lab <CISL>`_, in particular
+the Cheyenne, Geyser, and Caldera clusters.
+NCAR's `Research Data Archive`_ catalogs a huge number of datasets available
+from those clusters via the GLADE file systems.
+All of those datasets are accessible to Pangeo now via the Cheyenne deployment.
+
+Specific use case examples which load data from HPC platforms include:
+
+- Sean's thing
+- What else?
+
+Data in the Cloud
+-----------------
+
+The Pangeo community is very excited about the opportunities presented by the
+cloud for scientific research in terms of scalability, reproducibility, and
+efficiency.
+However, the cloud presents new challenges; the data formats that work well
+on personal computers and HPC systems don't necessarily translate well to the
+cloud environment.
+For an overview of these challenges, we recommend Matt Rocklin's excellent
+blog post:
+
+- `HDF in the Cloud: challenges and solutions for scientific data <http://matthewrocklin.com/blog/work/2018/02/06/hdf-in-the-cloud>`_
+
+The Pangeo community is actively engaged in defining what constitutes
+"analysis ready geoscience data" on the cloud.
+We are experimenting with many different technologies and options.
+This is a rapidly evolving area.
+If you have ideas on this topic you would like to share with our community,
+please  `reach out to us on Github <https://github.com/pangeo-data/pangeo/issues>`_.
+
+Our *current preference* for storing multidimensional array data in the cloud
+is the Zarr_ format.
+Zarr is a new storage format which, thanks to its simple yet well-designed
+specification, makes large datasets easily accessible to distributed computing.
+In Zarr datasets, the arrays are divided into chunks and compressed.
+These individual chunks can be stored as files on a filesystem or as objects
+in a cloud storage bucket.
+The metadata are stored in lightweight ``.json`` files.
+Zarr works well on both local filesystems and cloud-based object stores.
+Existing datasets can easily be converted to zarr via
+`xarray's zarr functions <http://xarray.pydata.org/en/latest/io.html#zarr>`_.
+
+Google Cloud Storage Data Catalog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Thanks to our :ref:`NSF Award <nsf-award>`, Pangeo has a substantial allocation
+on Google Cloud Platform which we are currently using to host several large
+datasets.
+These datasets are directly accessible from the Google Cloud Pangeo deployment:
+`pangeo.pydata.org <http://pangeo.pydata.org>`_.
+
+**HOW DO WE LIST THE DATASETS? CAN WE DIRECTLY PARSE THE INTAKE CATALOG?**
+
+
+.. _cloud-data-guide:
+
+Guide to Preparing Cloud-Optimized Data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+How do you get data into the cloud in a format that is optimized for pangeo
+to read?
+This guide outlines our current practices for converting netCDF data to
+Zarr_ and placing it on the cloud.
+These recommendations may change as cloud storage technology evolves.
+
+.. note::
```

Comment:
This is rendering as plain text on GitHub, not sure if it's working properly when using Sphinx.

Reply:
All looks good when I build locally.